tee_supplicant/include/optee_msg_supplicant.h

```diff
@@ -0,0 +1,300 @@
+/*
+ * Copyright (c) 2016-2017, Linaro Limited
```

Comment:
Please add 
Copyright (c) 2023 EPAM Systems

Reply:
There is no our corrections

Reply:
How can I see it from the history?

Reply:
Excuse me... Which history? Here is no history of original OP-TEE client. In this repository this is first commit with this file. And again. I don't want to put our copyright if we didn't change this file at all.

Reply:
At least it should be original files and then your changes in a separate commit. As for the "Which history?" question - you build up history. And for my vision - if you take file from somewhere everybody should be able to see what particular changes you've made for this file. That what history is for.

Reply:
I put this file to separate commit

Reply:
thanks.